server/utils/placementRequests/mapTaskForSummaryList.ts

```diff
@@ -0,0 +1,20 @@
+import { ApprovedPremisesApplication as Application, PlacementApplication } from '../../@types/shared'
+import { SummaryListWithCard } from '../../@types/ui'
+import PlacementApplicationForm from '../../form-pages/placement-application'
+import { forPagesInTask } from '../applications/forPagesInTask'
+import { mapPageForSummaryList } from './checkYourAnswersUtils'
+
+export const mapTaskForSummaryList = (
+  placementApplication: PlacementApplication,
+  application: Application,
+): Array<SummaryListWithCard> => {
+  const pageSummaries: Array<SummaryListWithCard> = []
+  const placementApplicationTask = PlacementApplicationForm.sections[0].tasks[0]
+  delete placementApplicationTask.pages['check-your-answers']
```

Comment:
Do we need to delete `check-your-answers` here, as `forPagesInTask` excludes it?

Reply:
Yeah you're right, this was introduced before I realised it's better to exclude it at `forPagesInTask`